tests/functional/restart/58-removed-task.t

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+
+GitHub #5067: if a task is removed from the graph after shutdown, it should not
+cause an error at restart. If it was a failed incomplete task, however, it
+should still be polled and logged at restart.
```

Comment:
```suggestion
# GitHub 5067: if a task is removed from the graph after shutdown, it should not
# cause an error at restart. If it was a failed incomplete task, however, it
# should still be polled and logged at restart.
```

Reply:
Argghhhhh! Fail!  :cry: :rofl: :face_with_spiral_eyes: :-1: :man_facepalming:  I added the comment at the last minute before pushing the final commit to the branch and leaving work - that'll teach me.